### ISSUE TITLE
Changes to Taxonomy Search Page need to be reflected in documentation

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/documentation-wiki#164]
These changes are in production! Now we need to update documentation.

_Originally posted by @Jegelewicz in https://github.com/ArctosDB/arctos/issues/3245#issuecomment-737309846_

**Comment by Jegelewicz** [comment]
Documents to update:
 - [x] https://www.youtube.com/watch?v=kHUJHa48m5E&feature=youtu.be
 - [x] Do we need a written how-to for using the search taxonomy page?

**Comment by sharpphyl** [comment]
Yes, we need to update it.  I can work on getting it done within a week or so if that's soon enough.

**Comment by Jegelewicz** [comment]
Anything you do and any time you do it will be soon enough! Thank you!!!!
